lib/rollbar.rb

```diff
@@ -21,6 +21,52 @@
 require 'rollbar/delay/thread'
 require 'rollbar/truncation'
 
+a = "foo bar"
+
+  a.upcase
+
+def long_method
+  environment = configuration.environment
+      environment = 'unspecified' if environment.nil? || environment.empty?
+
+      data = {
+        :timestamp => Time.now.to_i,
+        :environment => environment,
+        :level => level,
+        :language => 'ruby',
+        :framework => configuration.framework,
+        :server => server_data,
+        :notifier => {
+          :name => 'rollbar-gem',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.